scripts/ldapSync.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+set -eu
+
+default_time_out=39600
+value_time_out=${SERVER_LDAP_SYNC_FULL_CRONJOB_TIMEOUT:-$default_time_out}
+
+default_api_key="example"
+value_api_key="${SYNC_API_KEY:-$default_api_key}"
+
+# Start server in the background and redirect logs to file
+nohup npm run nest:start > server.log 2>&1 &
+
+# Show logs and wait for server to start up
+echo "Server logs:"
+tail -f server.log &
+
+echo "Wait for server to start..."
+until curl -s -w --retry 360 --retry-connrefused --retry-delay 10 http://localhost:3030/serverversion >/dev/null 2>&1; do
```

Comment:
serverversion is not running in any environment, but only on K8
therefor, the script can't be running on local, right? if so, I would avoid putting it in scripts folder. Maybe in ansible folder?

Reply:
as the serverversion can simply be faked locally this is not an issue. otherwise you can "relatively easy" test it locally with https://docs.dbildungscloud.de/pages/viewpage.action?pageId=248349347
so I guess it could stay in scripts folder with the other scripts if thats ok for you